### DISCUSSION TITLE
Just upgraded to 1.22 - lost data for last 5 days?

### DISCUSSION THREAD

**Discussion by chesteraser** [teslamate-org/teslamate#1482] [Q&A] [answered]
Encountered the login auth problem that was fixed in 1.22. Upgraded from 1.19 to 22 and it looks like everything is updating correctly now, however all data for the last 5 days is missing. Is there a way to retrieve this info from the Telsa API and import it, or should teslamate have automagically done that when the login was authorized again?

**Comment by adriankumpf** [comment, marked as answer]
Unfortunately, data cannot be retrieved retrospectively.
